apps/api_banking_web/lib/api_banking_web/controllers/transaction_controller.ex

```diff
@@ -19,13 +19,33 @@ defmodule ApiBankingWeb.TransactionController do
     end
   end
 
+  def create_transfer(conn, params) do
+    with {:ok, input_params} <- InputValidation.cast_and_apply(params, Inputs.Transfer),
+         {:ok, transaction} <- ApiBanking.create_transfer(input_params) do
+      format_transaction(transaction)
```

Comment:
Acho que se tu botar o credo ele já vai lançar essas sugestões:
```suggestion
      transaction
      |> format_transaction()
```